flang/test/Fir/complex.fir

```diff
@@ -5,6 +5,7 @@
 // RUN: ./a.out | FileCheck %s --check-prefix=EXECHECK
 
 // REQUIRES: x86-registered-target
+// UNSUPPORTED: x86_64-apple-darwin, x86_64-apple-macos
```

Comment:
This could be simplified as:
```
! UNSUPPORTED: darwin, macos
```
See: https://llvm.org/docs/TestingGuide.html#constraining-test-execution

Reply:
Done.